pyriemann/classification.py

```diff
@@ -192,7 +186,7 @@ def predict_proba(self, X):
         prob : ndarray, shape (n_trials, n_classes)
             the softmax probabilities for each class.
         """
-        return softmax(-self._predict_distances(X))
+        return softmax(-self._predict_distances(X)**2)
```

Comment:
This is bug fix I suppose

Reply:
It is the fix from PR #100. I had started to merge the unmerged PRs I needed in the main branch of my fork. This should not be an issue because you already merged it.
At the time of this PR, I did not realize I made the changes on my main branch, so now it's a mess :) I can rebase on my side if you want.